evals/cybench/task.py

```diff
@@ -0,0 +1,42 @@
+from collections.abc import Callable
+from pathlib import Path
+from textwrap import dedent
+
+from dataset import create_dataset
+
+from inspect_ai import Task, task
+from inspect_ai.dataset import Sample
+from inspect_ai.scorer import includes
+from inspect_ai.solver import Plan, basic_agent, plan, system_message
+from inspect_ai.tool import bash, python
+
+
+@task
+def ctf(
+    challenges_dir: Path | list[Path] | None = None,
+    filters: Callable[[Sample], bool] | list[Callable[[Sample], bool]] | None = None,
+    plan: Plan | None = None,
```

Comment:
`plan: Plan` => `solver: Solver`

Reply:
I decided to go with `agent: Solver` but don't mind if we want to name it `solver` instead.

Reply:
I think `agent: Solver` is the way to go!